torchao/float8/float8_tensor.py

```diff
@@ -313,7 +313,12 @@ def __new__(
             linear_mm_config if linear_mm_config is not None else LinearMMConfig()
         )
         self._gemm_input_role = gemm_input_role
-        assert axiswise_dim in (None, 0, -1), f"unsupported axiswise_dim {axiswise_dim}"
+        assert axiswise_dim in (
+            None,
+            0,
+            -1,
+            -2,
```

Comment:
Need to allow -2 so we can do rowwise scaling of the 2D subtensors embedded in 3D tensors.

e.g.

- A = (M,K) and B = (B,K,N)
- A_scale should be (M,)
- B_scale should be (B,N) <-- because we computed rowwise scales for the (K,N) subtensors